pkg/service/influx.go

```diff
@@ -1,7 +1,22 @@
 package service
 
-import "github.com/influxdata/influxdb-client-go/v2/api/write"
+import (
+	"context"
+	"fmt"
+	"time"
 
-func (s *service) WriteNewDataPoint(p *write.Point) {
-	s.influxDBWriteClient.WritePoint(p)
+	"github.com/instill-ai/pipeline-backend/pkg/utils"
+)
+
+// TODO: better naming for this file
+
+func (s *service) WriteNewDataPoint(ctx context.Context, data utils.UsageMetricData) {
+
+	s.redisClient.RPush(ctx, fmt.Sprintf("user:%s:trigger.trigger_time", data.OwnerUID), data.TriggerTime.Format(time.RFC3339Nano))
```

Comment:
The key is conflict with model-backend datapoint. I think the data will mess up together.

Reply:
oh yes good catch, will update that